crates/pipeline_manager/src/config.rs

```diff
@@ -24,7 +24,7 @@ fn default_sql_compiler_home() -> String {
 }
 
 fn default_db_connection_string() -> String {
-    "postgres-embed".to_string()
+    "".to_string()
```

Comment:
Should this be postgres:// now or something similar?

Reply:
tokio-postgres doesn't fill things in automatically, so "postgres://" does not work.